TrackerAlignment/scripts/alignment.py

```diff
@@ -0,0 +1,139 @@
+from sympy import Symbol, Matrix, diff, sqrt, atan2, cos, sin
```

Comment:
Where does this python code get used?  Is it diagnostic or part of the alignment procedure?

Reply:
It's part of the analytical derivatives. I thought it'd be good to separate out the algebra + maths that would be common to any kind of track from the code-generation logic specific to CosmicReco-type tracks (`generate_derivatives.py`), which imports this file.